v19.1/cost-based-optimizer.md

```diff
@@ -27,38 +27,20 @@ The most important factor in determining the quality of a plan is cardinality (i
 
 ## View query plan
 
-To see whether a query will be run with the cost-based optimizer, run the query with [`EXPLAIN (OPT)`](explain.html). The `OPT` option displays a query plan tree, along with some information that was used to plan the query. If the query is unsupported (i.e., it returns an error message that starts with e.g., `pq: unsupported statement` or `pq: aggregates with FILTER are not supported yet`), the query will not be run with the cost-based optimizer and will be run with the legacy heuristic planner.
+To see whether a query will be run with the cost-based optimizer, run the query with [`EXPLAIN (OPT)`](explain.html#opt-option). The `OPT` option displays a query plan tree, along with some information that was used to plan the query.
```

Comment:
Can we emphasize that `explain (opt)` isn't really intended for use by users (except to detect if a plan is running with CBO vs. HP)? The common case should be that they just use `explain`, and if that's missing information users need we should add it there.

Reply:
> Can we emphasize that explain (opt) isn't really intended for use by users (except to detect if a plan is running with CBO vs. HP)?

I think I've achieved roughly this by removing all of the bulleted list items that describe the different `EXPLAIN (OPT)` variants, ending the section with the "if a query is unsupported" statement since that's the use case we want to emphasize.

I didn't add more content in the spirit of "in all other cases, just use EXPLAIN" since it would spend more words to say "use this feature in this one case but not otherwise" which would paradoxically serve to emphasize it even more, I think. :-)

FTR these options are documented over at [`EXPLAIN`](https://www.cockroachlabs.com/docs/dev/explain.html#opt-option), but extremely briefly - mostly just to note their existence.